packages/generator-helper/src/dmmf.ts

```diff
@@ -86,6 +87,12 @@ export namespace DMMF {
      * BigInt, Boolean, Bytes, DateTime, Decimal, Float, Int, JSON, String, $ModelName
      */
     type: string
+    /**
+     * Native database type, if specified.
+     * For example, `@db.VarChar(191)` is encoded as `['VarChar', ['191']]`,
+     * `@db.Text` is encoded as `['Text', []]`.
+     */
+    nativeType?: [string, string[]] | null
```

Comment:
@pasha-vuiko I marked it as optional in the types even though the engine currently technically always serializes it as a value or null to future proof the types if we want to skip serializing it if it's `None`